tests/robustness/robustness_test/main_test.go

```diff
@@ -63,7 +63,7 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		// Clean the temporary dirs from the file system, don't write out the
 		// metadata, in case there was an issue loading it
-		eng.CleanComponents()
+		eng.CleanAllComponents()
```

Comment:
What about the call to `eng.Cleanup(index)` on line 77? It is followed by a `log.Fatalln` and a redundant `panic`.